README.md

```diff
@@ -62,6 +62,28 @@ scm-source
 docker build --tag <pierone_related_address>
 ```
 
+## Authentication
+
+### Enabling OAuth authentication
+
+OAuth is enabled by default for production (prod profile) or optionally when a oauth profile has been specified.
+
+```
+-Dspring.profiles.active=dev,oauth
+```
+
+### Acquiring token for testing
+
+For testing purpose is convenient to acquire token from Zalando OAuth2 token server. This can be easily done using
+`zign` in order to install it follow this steps:
+
+```bash
+sudo pip3 install --upgrade stups
+sudo pip3 install --upgrade stups-zign
+stups configure stups.zalan.do
+zign token
```

Comment:
Sorry for not mentioning it earlier. Most probably we need to update that as well since STUPS changed a bit. Maybe you can briefly check the STUPS training preparation document.